### DISCUSSION TITLE
When will the next release build/android 13 build release?

### DISCUSSION THREAD

**Discussion by notroger7** [LawnchairLauncher/lawnchair#3501] [Q&A] [answered]
It's hard to believe it is already half a year since the last lawnchair release on Android 12. Android 14 is already out for beta, while Lawnchair hasn't been updated to Android 13. When can we expect such a build to come out? 

Also, when can we expect a full stable release on the play store?

**Comment by SuperDragonXD** [comment, marked as answer]
Hello! I know that this reply is a bit late, but we are now on Lawnchair 15 *and* have also released a Play Store version of Lawnchair/Lawnicons.
